src/main/java/frc/trigon/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,113 @@
+package frc.trigon.robot.subsystems.arm;
+
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix6.configs.CANcoderConfiguration;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.signals.AbsoluteSensorRangeValue;
+import com.ctre.phoenix6.signals.SensorDirectionValue;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class ArmConstants {
+    private static final int
+            MASTER_ANGLE_MOTOR_ID = 0,
+            FOLLOWER_ANGLE_MOTOR_ID = 1,
+            MASTER_ELEVATOR_MOTOR_ID = 2,
+            FOLLOWER_ELEVATOR_MOTOR_ID = 3,
+            ELEVATOR_MAG_ENCODER_ID = 4,
+            ANGLE_CANCODER_ID = 5;
+
+    private static final double
+            ELEVATOR_MAG_ENCODER_OFFSET = 0.0,
+            ANGLE_CANCODER_OFFSET = 0.0;
+
+    private static final boolean
+            MASTER_ANGLE_INVERTED_VALUE = false,
+            FOLLOWER_ANGLE_INVERTED_VALUE = false,
+            MASTER_ELEVATOR_INVERTED_VALUE = false,
+            FOLLOWER_ELEVATOR_INVERTED_VALUE = true,
+            ELEVATOR_MAG_ENCODER_PHASE_VALUE = true;
+    private static final SensorDirectionValue ANGLE_CANCODER_SENSOR_DIRECTION = SensorDirectionValue.Clockwise_Positive;
+
+    private static final double
+            P = 1.32,
+            I = 3.1,
+            D = 0;
+    private static final PIDController PID_CONTROLLER = new PIDController(P, I, D);
+
+    private static final CANSparkMax.IdleMode
+            ANGLE_NEUTRAL_MODE_VALUE = CANSparkMax.IdleMode.kBrake,
+            ELEVATOR_NEUTRAL_MODE_VALUE = CANSparkMax.IdleMode.kBrake;
+    private static final AbsoluteSensorRangeValue ANGLE_ENCODER_SENSOR_RANGE = AbsoluteSensorRangeValue.Signed_PlusMinusHalf;
+
+    private static final CANSparkMax
+            MASTER_ANGLE_MOTOR = new CANSparkMax(MASTER_ANGLE_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            FOLLOWER_ANGLE_MOTOR = new CANSparkMax(FOLLOWER_ANGLE_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            MASTER_ELEVATOR_MOTOR = new CANSparkMax(MASTER_ELEVATOR_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless),
+            FOLLOWER_ELEVATOR_MOTOR = new CANSparkMax(FOLLOWER_ELEVATOR_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private static final TalonSRX ELEVATOR_MAG_ENCODER = new TalonSRX(ELEVATOR_MAG_ENCODER_ID);
+    private static final CANcoder ANGLE_CANCODER = new CANcoder(ANGLE_CANCODER_ID);
```

Comment:
Don't say `MAG_ENCDOER` or `CANCODER`, same way you wouldn't say NEO. Just type `ENCODER`. Change this for all the constants above as well